rc2/tests/test_vectors.rs

```diff
@@ -0,0 +1,38 @@
+
+pub struct RC2CipherTest {
+    pub key: &'static [u8],
+    pub eff_key_length: usize,
```

Comment:
I think it's better to store tests as files and use `new_block_cipher_tests`, so code will look something like this:
```Rust
#[test]
fn rc2_effective_key_64() {
    let mut buf = GenericArray::new();
    let tests = new_block_cipher_tests!("3", "4", "5", "6");
    for test in tests {
        let cipher = rc2::RC2::new_with_effective_key_length(test.key, 64);

        cipher.encrypt_block(&from_slice(test.input), &mut buf);
        assert_eq!(test.output, &buf[..]);
        cipher.decrypt_block(&from_slice(test.output), &mut buf);
        assert_eq!(test.input, &buf[..]);
    }
}
```

Same for 129 bits. This way all tests will be stored in the same format. If you don't want to deal with the boilerplate, you can write a simple macro. Also you can add sub-folders, say, `data/eff_key_64/` and `data/eff_key_129/` if you want to distinguish different types of tests.

Also you missed test with the single byte key. (number 4)